data_validation/cli_tools.py

```diff
@@ -641,6 +740,75 @@ def _add_common_arguments(parser):
     )
 
 
+def _add_common_partition_arguments(optional_arguments, required_arguments=None):
+    """Add all arguments common to get-partition command"""
+
+    # Group all Required Arguments together
+    required_arguments.add_argument(
+        "--source-conn", "-sc", required=True, help="Source connection name"
+    )
+    required_arguments.add_argument(
+        "--target-conn", "-tc", required=True, help="Target connection name"
+    )
+    required_arguments.add_argument(
+        "--config-dir",
+        "-cdir",
+        required=True,
+        help="Directory Path to store YAML Config Files",
+    )
+    required_arguments.add_argument(
+        "--partition-num",
+        "-pn",
+        required=True,
+        help="Number of partitions/config files to generate",
+        type=int,
+        choices=range(1, 1001),
+        metavar="[1-1000]",
+    )
+
+    # Optional arguments
+    optional_arguments.add_argument(
```

Comment:
Don't need flags from line 770 - 801 as mentioned above

Reply:
Removed from readme.md but kept required ones in cli_tools.py to build ConfigManager object